### ISSUE TITLE
[Bug]: the xfce pulseaudio plugin not working in Debian

### ISSUE THREAD

**Issue by mhdtyb** [termux/proot-distro#415]
### Problem description

i am on Android 14,using Debian with xfce and Termux:X11 in proot-distro, i installed all required packages in termux and in proot-distro, i do all required steps, but when i runned the gui everything works exept pulseaudio.
pulseaudio worked in termux (Native GUI in termux), and worked in ubuntu proot-distro, but not worked in Debian
![Screenshot_2024-06-06-14-56-08-38_84d3000e3f4017145260f7618db1d683](https://github.com/termux/proot-distro/assets/161652353/29f28d2c-e1fc-4e34-bab3-d3223ee05ef6)

### What steps will reproduce the bug?

running xfce in Debian proot-distro using Termux:x11

**Comment by reeaosirjustsir** [comment]
Don't worry! **I gotcha you!**

1. Create another termux session
2. Go another termux session
3. Type `apt update &>/dev/null; apt upgrade -y; pkg install pulseaudio -y` and enter
4. Type `nano $PREFIX/etc/pulse/default.pa` and press enter
5. Find other line and type **"load-module module-native-protocol-tcp auth-ip-acl=127.0.0.1 auth-anonymous=1"**
6. Save and exit default.pa
7. Type `nano  $PREFIX/etc/pulse/daemon.conf` and enter
8. Search **"exit-idle-time"** line (not **"quit-idle-time"** line) and replace "20" number with "-1" number
9. Save and exit daemon.conf
10. Try restart your proot debian desktop
10a. If not, join termux's discord server in 2 days (because that why i gets timeout yesterday):
https://discord.com/invite/termux-641256914684084234

The solution is made for **technical bot**:
https://youtu.be/CbEe-rE8MM4?si=4l9c5qsc_Oc4H_jB

**Comment by reeaosirjustsir** [comment]
> Yes, you can't use Pulseaudio provided in distributions. Need to start it in Termux and provide access via TCP, although I heard that binding directory with unix socket could work too.

![giphy](https://github.com/termux/proot-distro/assets/170645034/0dde6fd4-dce4-4d8c-993b-b1b862e01d75)

Are your solution is **fortunately better** than my solution?

**Comment by twaik** [comment]
@reeaosirjustsir github is not discord. Please avoid posting such gifs here.

**Comment by mhdtyb** [comment]
> Don't worry! **I gotcha you!**
> 
> 1. Create another termux session
> 2. Go another termux session
> 3. Type `apt update &>/dev/null; apt upgrade -y; pkg install pulseaudio -y` and enter
> 4. Type `nano $PREFIX/etc/pulse/default.pa` and press enter
> 5. Find other line and type **"load-module module-native-protocol-tcp auth-ip-acl=127.0.0.1 auth-anonymous=1"**
> 6. Save and exit default.pa
> 7. Type `nano  $PREFIX/etc/pulse/daemon.conf` and enter
> 8. Search **"exit-idle-time"** line (not **"quit-idle-time"** line) and replace "20" number with "-1" number
> 9. Save and exit daemon.conf
> 10. Try restart your proot debian desktop
>     10a. If not, join termux's discord server in 2 days (because that why i gets timeout yesterday):
>     https://discord.com/invite/termux-641256914684084234
> 
> The solution is made for **technical bot**: https://youtu.be/CbEe-rE8MM4?si=4l9c5qsc_Oc4H_jB

yeah i know but why it worked with me (sound worked) in ubuntu without enabling pulseaudio server in termux ?
